v4/feature/market/src/main/java/exchange/dydx/trading/feature/market/marketinfo/components/position/DydxMarketPositionButtonsViewModel.kt

```diff
@@ -82,6 +82,12 @@ class DydxMarketPositionButtonsViewModel @Inject constructor(
                 orders = stopLossOrders,
                 configsAndAsset = configsAndAsset,
             ),
+            editMarginAction = {
+                router.navigateTo(
+                    route = TradeRoutes.adjust_margin + "/$marketId",
+                    presentation = DydxRouter.Presentation.Modal,
+                )
+            },
```

Comment:
do u also need to pass childSubaccountNumber # ?

Reply:
No... It's using the abacusStateManager.subaccountNumber to get the subaccountNumber.  From there, we can get the selected subaccount.  The subaccount's openPosition of the market should have the childSubaccountNumber.